nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -37,11 +37,19 @@ in
     environment.systemPackages = [ cfg.package ];
 
     environment.etc."wireplumber/main.lua.d/80-nixos.lua" = lib.mkIf (!pwUsedForAudio) {
-     text = ''
+      text = ''
```

Comment:
Extra space snuck in here?

Reply:
So sad this PR got merged with this extra space.

Reply:
seems intentional and correct

Reply:
Indeed. Apparently this just looked weird in my browser. I've just checked it locally.

Reply:
Indeed it was intentional. I copy/pasted this block and noticed it was off. Thanks for the quick review.